include/crm/common/ordering.h

```diff
@@ -25,11 +25,14 @@ enum pe_ordering {
     //! Ordering disabled (compare with equality rather than bit set)
     pcmk_no_ordering                    = 0U,
 
+    //! Ordering enabled (and optional, if no other flags are set)
+    pcmk_ord_enabled                    = (1U << 0),
```

Comment:
Yeah and it's worse since the flag is not required to enable the ordering, another flag is usually sufficient. One day I'd love to overhaul the ordering code (again) to have more intuitive combinations of flags, but that will be backward-incompatible.